### PR TITLE
 Increases the inventory of vendors with additional items and a greater quantity of goods.

### PR DESCRIPTION
**Pull request by HaultyAnonie**
Changes the vendors so less people die in a round and stay playing.

## About the Pull Request

 Increases the inventory of vendors with additional items and a greater quantity of goods **pretty much sums it up.**

## Why It's Good For The Game

These changes benefit the game, especially during the uptimes we have less people dying and more people testing; this also would account for future higher-pop if all the things in the discord's channel 'showcase' would be implemented and we get more developers.

## Changelog

:cl:
add: body bags to medical and security vendors
tweak: tweaked the description and announcement says of the vendors.
tweak: tweaked the name of the wall-mounted NanoMed.
tweak: tweaked the amount of injectors, pouches and other items a vendor holds to be higher.
add:  comments to clarify some injectors.
/:cl:
# Before
image
image
image
# After
image
image
image

### CODE REVIEW

File: code/game/machinery/vending.dm [hunk 1]
```diff
@@ -962,28 +963,29 @@
 
 /obj/machinery/vending/wallmed1
 	name = "\improper NanoMed"
-	desc = "A wall-mounted version of the NanoMed."
+	desc = "A wall-mounted version of the NanoMed, stocked with the equipment necessary for stabilisation and much more."
```
Comment (TheDarkElites) [inline code/game/machinery/vending.dm:966]:
```suggestion
	desc = "A wall-mounted version of the NanoMed, stocked with the equipment necessary for stabilization and much more."
```

File: code/game/machinery/vending.dm [hunk 2]
```diff
@@ -1003,21 +1008,23 @@
 
 /obj/machinery/vending/security
 	name = "\improper SecTech"
-	desc = "A security equipment vendor."
-	product_ads = "Crack capitalist skulls!;Beat some heads in!;Don't forget - harm is good!;Your weapons are right here.;Handcuffs!;Freeze, scumbag!;Don't tase me bro!;Tase them, bro.;Why not have a donut?"
+	desc = "A security equipment vendor, with some customary donuts."
+	product_ads = "Defend yourself!;Help others!;All you need for a secure facility right here!;Your weapons are right here.;Handcuffs!;Did you know that 'Freeze, scumbag!' is the most standardised quote broadcasted by this machine!;Try your best to apprehend all criminals and subject them to the law.;Reminder to always use non-lethals first!;Why not have a donut?"
```
Comment (TheDarkElites) [inline code/game/machinery/vending.dm:1012]: should probably also leave Freeze, scumbag in by itself alongside the modified one as that would be funny.

File: code/game/machinery/vending.dm [hunk 3]
```diff
@@ -1003,21 +1008,23 @@
...
 	icon_state = "sec"
 	icon_deny = "sec-deny"
 	icon_vend = "sec-vend"
 	vend_delay = 14
 	base_type = /obj/machinery/vending/security
-	req_access = list(ACCESS_SECURITY)
+	req_access = list(ACCESS_SECURITY_LVL1)
```
Comment (TheDarkElites) [inline code/game/machinery/vending.dm:1018]: please double check that this still allows all sec to use it, the way our access is handled is a bit shit and needs to be reworked so its possible this only allows for level one sec guards to access this.

File: code/game/machinery/vending.dm (hunk 1 above)
Comment (TheDarkElites) [inline code/game/machinery/vending.dm]:
```suggestion
	desc = "A wall-mounted version of the NanoMed, stocked with the equipment necessary for stabilization and much more."
```

File: code/game/machinery/vending.dm (hunk 2 above)
Comment (TheDarkElites) [inline code/game/machinery/vending.dm]: Should leave in "Freeze Scumbag" alongside the modified version you made cause its funny then.

**Review by TheDarkElites — CHANGES_REQUESTED** [review]

File: code/game/machinery/vending.dm (hunk 3 above)
Comment (HaultyAnonie) [reply code/game/machinery/vending.dm:1018]: it allows.

File: code/game/machinery/vending.dm (hunk 2 above)
Comment (HaultyAnonie) [reply code/game/machinery/vending.dm]: Suggest it and I'll commit it with you as a co-author I'm too tired.

Comment (HaultyAnonie) [reply code/game/machinery/vending.dm]: Is it not already in there... wut.

Comment (TheDarkElites) [reply code/game/machinery/vending.dm]: no its not, you edited the original one

Comment (HaultyAnonie) [reply code/game/machinery/vending.dm]: Okay, I'll do that.


> no its not, you edited the original one

**Pull request closed without merging**